corehq/apps/ota/migrations/0002_alter_db_index.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ota', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
This can even be in the same migration, just as a separate operation. Also fine to leave it separate.